src/main/java/com/google/cloud/spanner/pgadapter/wireprotocol/DescribeMessage.java

```diff
@@ -87,13 +87,21 @@ public void handleDescribePortal() throws Exception {
     if (this.statement.hasException()) {
       this.handleError(this.statement.getException());
     } else {
-      new RowDescriptionResponse(
-              this.outputStream,
-              this.statement,
-              ((DescribePortalMetadata) this.statement.describe()).getMetadata(),
-              this.connection.getServer().getOptions(),
-              QueryMode.EXTENDED)
-          .send();
+      switch (this.statement.getResultType()) {
+        case UPDATE_COUNT:
+        case NO_RESULT:
+          new NoDataResponse(this.outputStream).send();
```

Comment:
PostgreSQL returns `NoData` as as response to a `Describe Portal` for an update count or other statement that does not return a result set.